lib/time_for_a_boolean.rb

```diff
@@ -4,17 +4,22 @@
 require "time_for_a_boolean/railtie"
 
 module TimeForABoolean
-  def time_for_a_boolean(attribute)
+  def time_for_a_boolean(attribute, backing_attribute_name=nil)
+    backing_attribute_name ||= "#{attribute}_at"
```

Comment:
![](http://dl.dropboxusercontent.com/u/86731380/gifs/mind_blow.gif)

I did not know you could reference the first argument in the second argument!